app/views/steps/respondent/personal_details/edit.html.erb

```diff
@@ -1,35 +1,33 @@
 <% title t('.page_title') %>
+<% step_header %>
 
-<div class="grid-row">
-  <div class="column-two-thirds">
-    <%= step_header %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= govuk_error_summary %>
 
-    <h1 class="heading-xlarge gv-u-heading-xxlarge"><%=t '.heading', name: @form_object.record.full_name %></h1>
+    <h1 class="govuk-heading-xl"><%=t '.heading', name: @form_object.record.full_name %></h1>
 
-    <%= step_form @form_object do |f| %>
-      <%=
-        f.radio_button_fieldset :has_previous_name, inline: true do |fieldset|
-          fieldset.radio_input GenericYesNoUnknown::YES, panel_id: :previous_name_panel
-          fieldset.radio_input GenericYesNoUnknown::NO
-          fieldset.radio_input GenericYesNoUnknown::UNKNOWN
-          fieldset.revealing_panel(:previous_name_panel) { |panel| panel.text_field :previous_name }
-        end
-      %>
+    <%= step_form @form_object, builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
+      <%= f.govuk_radio_buttons_fieldset :has_previous_name, legend: { tag: 'span', size: 'm' } do %>
+        <%= f.govuk_radio_button :has_previous_name, GenericYesNoUnknown::YES, link_errors: true do %>
+          <%= f.govuk_text_field :previous_name %>
+        <% end %>
+        <%= f.govuk_radio_button :has_previous_name, GenericYesNoUnknown::NO %>
+        <%= f.govuk_radio_button :has_previous_name, GenericYesNoUnknown::UNKNOWN %>
+      <% end %>
 
-      <%= f.radio_button_fieldset :gender, choices: Gender.values %>
-      <%= f.gov_uk_date_field :dob, placeholders: true, legend_text: t('shared.form_elements.dob'), form_hint_text: t('shared.form_elements.dob_hint') %>
+      <%= f.govuk_collection_radio_buttons :gender, Gender.values, :value, nil, legend: { tag: 'span', size: 'm' } %>
 
-      <%=
-        f.check_box_fieldset :dob_unknown, [:dob_unknown] do |fieldset|
-          fieldset.check_box_input(:dob_unknown) { f.text_field :age_estimate }
-        end
-      %>
+      <%= f.govuk_date_field :dob, legend: { tag: 'span', size: 'm' } %>
```

Comment:
We need to make the changes to the form object for this date to work, like with Applicants.